components/dashboard/src/SwitchToPAYG.tsx

```diff
@@ -429,35 +441,41 @@ function SwitchToPAYG() {
     const planDescription = pageState.old?.planDetails || "";
     const selectorEntries = getOrganizationSelectorEntries(teams || [], setSelectedOrganization);
     return (
-        <div className="flex flex-col max-h-screen max-w-3xl mx-auto items-center w-full mt-24">
-            <h1>{`Update your ${titleModifier}`}</h1>
-            <div className="w-full text-gray-500 text-center">
-                Switch to the new pricing model to keep uninterrupted access and get <strong>large workspaces</strong>{" "}
-                and <strong>custom timeouts</strong>.{" "}
+        <div className="flex flex-col max-h-screen max-w-2xl mx-auto items-center mt-24">
+            <Heading1>{`Update your ${titleModifier}`}</Heading1>
+            <Subheading className="w-full mt-3 text-gray-500 text-center">
+                Switch to the new pricing model to keep uninterrupted access and <br /> get{" "}
+                <strong>large workspaces</strong> and <strong>custom timeouts</strong>.{" "}
                 <a
                     className="gp-link"
                     href="https://www.gitpod.io/blog/introducing-workspace-classes-and-flexible-pricing"
```

Comment:
**suggestion**: It's worth opening this in a new window, right?

```suggestion
                    href="https://www.gitpod.io/blog/introducing-workspace-classes-and-flexible-pricing"
                    target="_blank"
```

Reply:
done